corehq/apps/cleanup/management/commands/delete_forms.py

```diff
@@ -0,0 +1,52 @@
+from django.core.management.base import BaseCommand
+import csv
+import itertools
+from dimagi.utils.chunked import chunked
+from corehq.form_processor.models import XFormInstance
+
+
+INDEX_FORM_ID = 1
+CHUNK_SIZE = 100
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('domain', help='Domain name that owns the forms to be deleted')
+        parser.add_argument('filename', help='path to the CSV file')
+        parser.add_argument('--resume_id', help='form ID to start at, within the CSV file')
+
+    def handle(self, domain, filename, resume_id=None, **options):
+        # expects the filename to have a CSV with a header containing "Deletion Status" and "Form ID" fields
+        with open(filename) as csvfile:
+            reader = csv.reader(csvfile, delimiter=',')
+            self._process_rows(reader, domain, resume_id)
+
+    def _process_rows(self, rows, domain, resume_id):
+        next(rows)   # skip header line
+
+        num_deleted = 0
+
+        if resume_id:
+            print('resuming at: ', resume_id)
+            rows = itertools.dropwhile(lambda row: row[INDEX_FORM_ID] != resume_id, rows)
+
+        print('Starting form deletion')
+        for chunk in chunked(rows, CHUNK_SIZE):
+            form_ids = [row[INDEX_FORM_ID] for row in chunk]
+
+            try:
+                deleted_form_ids = set(XFormInstance.objects.hard_delete_forms(
```

Comment:
I feel squeemish about hard-delete, though obviously it is what we're being asked to do. An alternate pattern would be to soft delete and mark for automatic hard-deletion in 90 days. But that is probably overkill for the present situation. Maybe what I would ask is just to rename the command to `hard_delete_forms.py`, since in some places delete means soft-delete, and making that name explicit will remove any kind of ambiguity for anyone reusing this script.

Reply:
Good point, I can definitely do that

Reply:
Addressed with 4fccff9483682f5ae3f13ae902db5b8fa069ccbe